File: benchmark/results/python-invocation-script/run-001/a-mcp/output/analytics/services/metrics_aggregator.py
from typing import List, Optional
from analytics.aggregates.event import Event

"""MetricsAggregator service."""
class MetricsAggregator:
    def create(self, input: Event) -> Event:
        raise NotImplementedError('not implemented')
    def findById(self, id: str) -> Optional[Event]:
        raise NotImplementedError('not implemented')
    def list(self, limit: int) -> List[Event]:
        raise NotImplementedError('not implemented')
    def delete(self, id: str) -> None:
        raise NotImplementedError('not implemented')

File: benchmark/results/ts-invocation-heredoc/run-005/b-baseline/output/src/domain/analytics/services/MetricsAggregator.ts
import { Event } from "../aggregates/Event";

/** Domain service for aggregating analytics metrics from events. */
export class MetricsAggregator {
  create(input: Partial<Event>): Event {
    void input;
    throw new Error("not implemented");
  }

  findById(id: string): Event | null {
    void id;
    throw new Error("not implemented");
  }

  list(limit: number): Event[] {
    void limit;
    throw new Error("not implemented");
  }

  delete(id: string): void {
    void id;
    throw new Error("not implemented");
  }
}
